.github/workflows/icu_ci.yml

```diff
@@ -32,6 +32,27 @@ jobs:
           ant init;
           ant check;
 
+  # Out of source build with gcc 8 and Wextra.
+  gcc-8-Wextra:
```

Comment:
Could we reasonably add the Wextra and hrdtst into another existing job, maybe `gcc-8-stdlib14`?
(And maybe upgrade the gcc version there if a newer one is available for GHA?)

Reply:
-Wextra added to job gcc-10-stdlib14.
hdrtst is also executed by job gcc-debug-build-and-test.